app/controllers/collection_drafts_controller.rb

```diff
@@ -414,4 +414,18 @@ def edit_view_setup
     set_data_centers          if @form == 'data_centers' || @form == 'data_contacts'
     load_data_contacts_schema if @form == 'data_contacts'
   end
+
+  def set_resource_by_model
+    set_resource(CollectionDraft.new(user: current_user, provider_id: current_user.provider_id, draft: {}))
+  end
+
+  def generate_model_error
+    return unless get_resource.errors.any?
+    model_error_message = ''
+    get_resource.errors.full_messages.each do |msg|
+      model_error_message += ', ' unless model_error_message == ''
+      model_error_message += msg.downcase
+    end
+    model_error_message
+  end
```

Comment:
This is going to get really ruby condensed (which some people don't like it as much), but can we try replacing lines 424 - 429 with:
`get_resource.errors.full_messages.reject{ |msg| msg.blank? }.map{ |msg| msg.downcase }.join(', ')`

Reply:
I would be more wary of it with larger datasets because you're walking it twice, but it is expected to be small here.

Reply:
Also rubocop suggests     get_resource.errors.full_messages.reject(&:blank).map(&:downcase).join(', ') instead